Scripts/estimateTotal.R

```diff
@@ -0,0 +1,78 @@
+library(data.table)
+
+sex <- fread("../Raw Data/OUDOrigin_Sex_26JUN2024.csv")
+race <- fread("../Raw Data/OUDOrigin_Race_26JUN2024.csv")
+age <- fread("../Raw Data/OUDOrigin_Twenty_26JUN2024.csv")
```

Comment:
Just a check to make sure we want these relative paths and exact file names in the code.

Reply:
File names are okay- data is not, so I think with `Raw Data` gitignored, it should be fine (and then reproducible for others// tracks lineage and data pull time)